packages/api/cms-api/src/auth/dto/current-user.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { Field, ObjectType } from "@nestjs/graphql";
+
+@ObjectType()
+export class CurrentUser {
```

Comment:
We need a way to make this extendable by the application. eg. to add allowed domains